src/workers/InlineWorker.js

```diff
@@ -5,18 +5,18 @@
 
 var URL = window.URL || window.webkitURL;
 
-function CreateWorker(url, workerType) {
+function CreateWorker(url, options, workerType) {
```

Comment:
Let's use `CreateWorker(workerType, url, options) { }` + we need update snapshots

Reply:
What do you mean update snapshots?